hedera-node/src/main/java/com/hedera/services/ServicesState.java

```diff
@@ -177,6 +177,9 @@ public void genesisInit(Platform platform, AddressBook addressBook, final Swirld
 		final var seqStart = bootstrapProps.getLongProperty("hedera.firstUserEntity");
 		createGenesisChildren(addressBook, seqStart);
 
+		networkCtx().setBlockNo(bootstrapProps.getLongProperty("blocks.lastBlockNumber"));
+		networkCtx().setLatestConsTimeOfCurrentBlock(bootstrapProps.getInstantProperty("blocks.lastBlockTimestamp"));
```

Comment:
Probably we can extract these two lines in a separate method - `initializeBlockProperties()`